src/Integration.UnitTests/State/StateManagerTests.cs

```diff
@@ -36,12 +36,6 @@ namespace SonarLint.VisualStudio.Integration.UnitTests.State
     [TestClass]
     public class StateManagerTests
     {
-        [TestInitialize]
-        public void TestInitialize()
-        {
-            ThreadHelper.SetCurrentThreadAsUIThread();
```

Comment:
The VS `ThreadHelper` is no longer used in the modified product code, so I've removed this call from any of the tests that had it.